mock/service/controller.go

```diff
@@ -326,6 +326,32 @@ func (s *service) ValidateVolumeCapabilities(
 	}, nil
 }
 
+func (s *service) ControllerGetVolume(
+	ctx context.Context,
+	req *csi.ControllerGetVolumeRequest) (
+	*csi.ControllerGetVolumeResponse, error) {
+
+	if hookVal, hookMsg := s.execHook("GetVolumeStart"); hookVal != codes.OK {
+		return nil, status.Errorf(hookVal, hookMsg)
+	}
+
+	i, v := s.findVolByID(ctx, req.VolumeId)
+	if i < 0 {
+		return nil, status.Error(codes.NotFound, req.VolumeId)
+	}
+
+	resp := &csi.ControllerGetVolumeResponse{
+		Volume: &v,
+		Status: &csi.ControllerGetVolumeResponse_VolumeStatus{},
```

Comment:
Could we generate the `VolumeCondition` field instead of returning an empty one (nil)?

Reply:
Please refer to the comments
```
type NodeGetVolumeStatsResponse struct {
	// This field is OPTIONAL.
	Usage []*VolumeUsage `protobuf:"bytes,1,rep,name=usage,proto3" json:"usage,omitempty"`
	// Information about the current condition of the volume.
	// This field is OPTIONAL.
	// This field MUST be specified if the VOLUME_CONDITION node
	// capability is supported.
	VolumeCondition      *VolumeCondition
...
```

Reply:
For error, I mock the VolumeCondition filed. For normal response, I mocked the PublishedNodeIds

Reply:
```       
        // Information about the current condition of the volume.
	// This field is OPTIONAL.
	// This field MUST be specified if the
	// VOLUME_CONDITION controller capability is supported.
```